src/main_nep/parameters.cu

```diff
@@ -1284,13 +1285,19 @@ void Parameters::parse_save_potential(const char** param, int num_param)
 {
   is_save_potential_set = true;
 
-  if (num_param != 2) {
-    PRINT_INPUT_ERROR("save_potential should have 1 parameter.\n");
+  if (num_param != 3) {
+    PRINT_INPUT_ERROR("save_potential should have 2 parameters.\n");
```

Comment:
should we set `save_potential_format=1` by default, and not enforce it to be explicitly set as is currently done?

Reply:
I have no strong opinion here as this is a new keyword that has not been widely used yet.

Reply:
Fair enough, lgtm then :+1: